nxdrive/engine/dao/migrations/manager/0004_initial_migration.py

```diff
@@ -0,0 +1,81 @@
+from sqlite3 import Cursor
+
+from ..migration import MigrationInterface
+
+
+class MigrationInitial(MigrationInterface):
+    def upgrade(self, cursor: Cursor) -> None:
+        """
+        Create all the basics table.
+        Setup the *journal_mode* and *temp_store*.
+        """
+        cursor.execute("PRAGMA journal_mode = WAL")
```

Comment:
Shouldn't be `DELETE` here? cf https://github.com/nuxeo/nuxeo-drive/blob/6aabec2119fd1cb1b8f0fdca9be984ebc6f9d165/nxdrive/engine/dao/manager.py#L22-L23